src/Http/Headers/src/HeaderNames.cs

```diff
@@ -61,6 +63,8 @@ public static class HeaderNames
     public static readonly string AltSvc = "Alt-Svc";
 
     /// <summary>Gets the <c>:authority</c> HTTP header name.</summary>
+    [Obsolete("For infrastructure use only", false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We shouldn't hide it from intellisense. That's one of the most annoying experiences as a developer when you see it in the docs and then it doesn't show up in the IDE. Obsoletion is enough